pages/entropy/fees.mdx

```diff
@@ -12,3 +15,25 @@ of the respective blockchain, ensuring a seamless and integrated operation.
 Note that protocols integrating with Entropy can pass these fees along to their users. Whenever a user
 submits a transaction that requests a random number, the user can directly pay the entropy fees required
 with the native blockchain token. There are no fees for revealing the random numbers.
+
+The following tables shows the total fees payable when using the default provider.
+
+## Mainnet
```

Comment:
I think these fee tables belong in the "Reference Material" section, not the "Understanding" section per the diataxis guidelines. Probably better to make a separate `Fee Schedule` page under Reference Material for that.